### PR TITLE
Add inclusive meeting printout link

### PR DESCRIPTION
**Pull request by enatario**
Added a public link to our inclusive meeting printout! Sorry for the mega-diff -- my editor formatted everything to an 80 character line length and used dashes in lieu of asterisks.

### CODE REVIEW

File: working-together/README.md [hunk 1]
```diff
@@ -16,50 +17,59 @@ A guide for working well together.
...
+- **Introduce everyone:** Don't assume everyone has met before. Introductions
+  help break the ice and give an opportunity for voices to be heard.
+- **Pay Attention:** Be present with others during a meeting, but also engage
+  and communicate in the way you feel most comfortable.
+- **Be respectful of others' schedules:** Try to start and end meetings on time.
+- **Step up, step back:** After you speak, let others speak. If you are
+  speaking often, make sure others have had a chance to say something.
+- **Apply the Shine Theory:** Notice if someone is being cut off or ignored. 
+Help
+  shine the light back on that contributor by repeating their idea, giving clear
```
Comment (louis-antonopoulos) [inline working-together/README.md]: I can't get the "Insert a suggestion" functionality to grab all three lines, even though all are highlighted. But `Help` looks lonely all on its own line...

File: working-together/README.md [hunk 2]
```diff
@@ -16,50 +17,59 @@ A guide for working well together.
...
+  "My feedback for that is..."
+- **Speak as I:** Try to speak from the _I_ perspective. Don't speak for others.
+- **Disagree:** You don't have to agree with everybody, but be aware of how you
+  frame your feedback. Criticize the idea, not the person.
+- **Everyone can participate:** Assuming someone doesn't want to participate
+  can be exclusionary. Give people the opportunity to decline an invitation.
+- **Speak up:** Follow up if team members treat others disrespectfully. If you
+  don't feel comfortable addressing someone directly, speak with your manager 
+or Chief
+  People Officer.
```
Comment (louis-antonopoulos) [inline working-together/README.md]: Same as above with the line breaks around `or Chief`

File: working-together/README.md (hunk 1 above)
Comment (enatario) [reply working-together/README.md]: Oh gosh yeah my formatter mucked this up a bit. Thanks for catching -- will fix!

**Comment by enatario** [comment]
@louis-antonopoulos should be all set now!

**Pull request merged**